openedx/stanford/common/djangoapps/enrollment/urls.py

```diff
@@ -1,15 +1,18 @@
 """
 URL for enrollment API
 """
+from django.conf import settings
 from django.conf.urls import patterns, url
 
-from .views import UpdateEnrollmentView
+from .views import EnrollmentStatusView
 
 urlpatterns = patterns(
     'enrollment.views',
     url(
-        r'^update_status$',
-        UpdateEnrollmentView.as_view(),
+        r"^enrollment/status/{course_key}/?$".format(
```

Comment:
Is the `status` namespace enough? Or do we need to explicitly put it behind `stanford/` something or other?

Reply:
I just want to make sure we don't step on edx's toes.. or get ours stepped on..
If we do something generic enough maybe it'll be easier to land upstream? 🤷‍♂️